docs/api-reference.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_label: API Reference
+sidebar_label: API Reference Yay!
```

Comment:
Yay for enthusiasm, but do we want a Yay here? :)

Reply:
lol I was showing Noa how to commit, somehow this ended here. removed.

Reply:
:)